vms/volunteer/views.py

```diff
@@ -41,7 +41,9 @@ def download_resume(request, volunteer_id):
                 filename = settings.MEDIA_ROOT + basename
                 wrapper = FileWrapper(file(filename))
                 response = HttpResponse(wrapper)
-                response['Content-Disposition'] = 'attachment; filename=%s' % os.path.basename(filename)
+                response[
+                    'Content-Disposition'] = 'attachment; filename=%s' % os.path.basename(

```

Comment:
this doesn't look good, what was the length w/ 45 being w/ 44? we can think of increasing the length to 85.

Reply:
All your suggested changes have been implemented. Please have look

Reply:
All your suggested changes have been implemented now. Most of the Flake8 errors have also been solved. It now also fixes the issue https://github.com/systers/vms/issues/379. Please have a look.

Reply:
Can you please resolve the conflicts too, you would see that `flake8 .` in `.travis.yml` is you need to add as it was recently removed.